lib/xdr/transactions/operations/invoke_host_function_op.ex

```diff
@@ -0,0 +1,73 @@
+defmodule StellarBase.XDR.InvokeHostFunctionOp do
```

Comment:
For the modules that are operations we need to:
- Add `Operations` to the namespace
- Remove suffix `Op` from the module name